### DISCUSSION TITLE
cannot translate to chinese lanaguage

### DISCUSSION THREAD

**Discussion by chungenhung** [openai/whisper#386] [Q&A] [answered]
Input audio file is in english, using python:
`result = model.transcribe("/home/user/whisper/english_short.aac", language="zh")`

Even though "zh" is in the list of languages in [https://github.com/openai/whisper/blob/9f70a352f9f8630ab3aa0d06af5cb9532bd8c21d/whisper/tokenizer.py](url) , but the output text simply show in english.

I have tested fr and ja language codes, and those transates ok.

**Comment by SutirthaChakraborty** [comment]
The 'Bengali' translation return string in 'Hindi'

**Comment by jianfch** [comment, marked as answer]
From the name of the file, assuming you're trying to translate English speech into Chinese text, this is the expected behavior. Whisper can do **transcription** in Chinese, but it only does Any-to-English speech translation.

**Reply by chungenhung** [reply]
Not true. as it can clearly translate from Eng to fr, ru, ja, etc.

**Reply by jianfch** [reply]
Perhaps, "only" is not exactly right. But it is only intended for X -> English. The `language` tag tells the model that X is the input language, and task is either X -> X (transcribe) or X->English (translate). Indeed, you can tell it that the input is French when the input is actually English then tell it to X -> X then it hallucinates and sometimes gets it right and "translates" English -> X. 
See [comment](https://github.com/openai/whisper/discussions/22#discussioncomment-3705465)
